### DISCUSSION TITLE
scikit-survival for osx-arm64

### DISCUSSION THREAD

**Discussion by nils-fl** [sebp/scikit-survival#206] [Q&A] [answered]
Hi,

I am one of these guys with an Apple M1 and very mixed feelings about it :D
Will `scikit-survival` be available for osx-arm64?

Cheers
Nils

**Comment by sebp** [comment, marked as answer]
I hope scikit-survival will be available for Apple Silicon at some point. I don't have access to an M1 myself, but it seems dependencies of scikit-survival are still working on adding support for M1:
- numpy: https://github.com/numpy/numpy/issues/17807
- scipy: https://github.com/scipy/scipy/issues/13409
- pandas: https://github.com/pandas-dev/pandas/issues/40611
- other dependencies that use compiled code: ecos, osqp

It looks like the upcoming numpy 0.12.1 release is going to support M1, so other projects should be able to start adding support for M1 soon.
